vignettes/model-card.Rmd

```diff
@@ -0,0 +1,76 @@
+---
+title: "Model cards for transparent, responsible reporting"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Model Cards for transparent, responsible reporting}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+
+## What is a "Model Card"?
+
+Good documentation helps us make sense of software, know when and how to use it, and understand its purpose. The same can be true of documentation or reporting for a deployed model, but it can be hard to know where to start. The paper ["Model Cards for Model Reporting" (Mitchell et al. 2019)](https://doi.org/10.1145/3287560.3287596) provides a suggested framework for organizing and presenting the essential facts about a deployed machine learning model. The vetiver package provides an R Markdown template for creating a "Model Card" for a published vetiver model. The template **automates** extracting some information from the model object, and provides **structure** for the model developer where automation is not possible.
+
+Model developers see a nudge to create a model card when they publish a model.
```

Comment:
Is it recommended to write a model card for each version of the model? Only deployed versions? Just the first one? It may be helpful to users to add context on cadence (if there is any).